projects/common/src/color/color.ts

```diff
@@ -1,7 +1,13 @@
 export const enum Color {
-  Purple2 = '#f2d0f5',
-  Purple5 = '#94209f',
+  Blue3 = '#70a7ff',
+  BlueGray4 = '#4b5f77',
```

Comment:
Do we want to call it Blue4 or does the mocks say it as BlueGray4?

Reply:
I picked this from the color palette. Color palette has blues and blue grays separate.